feature/metis/manage-conversations/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/manageconversations/ui/conversation/create_personal_conversation/PotentialRecipientsUi.kt

```diff
@@ -135,10 +143,32 @@ private fun PotentialRecipientsList(
             }
         }
     } else {
-        Box(modifier = modifier) {
+        val icon = if (isQueryTooShort) Icons.Default.KeyboardAlt else Icons.Default.PersonOff
+        val text = if (isQueryTooShort) {
+            stringResource(
+                id = R.string.conversation_member_selection_query_too_short,
+                MemberSelectionBaseViewModel.MINIMUM_QUERY_LENGTH
+            )
+        } else {
+            stringResource(id = R.string.conversation_member_selection_recipients_empty)
+        }
+
+        Column(
+            modifier = modifier,
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.spacedBy(8.dp, alignment = Alignment.CenterVertically),
```

Comment:
Why is the alignment necessary?

Reply:
Ah sorry, some how I did not send my answer to your previous comment.

I tried it out and without the alignment the column content is aligned to the top of the column. Having it centerred vertically looks much better imo